gpsd_client/src/client.cpp

```diff
@@ -18,7 +21,11 @@ namespace gpsd_client
       use_gps_time_(true),
       check_fix_by_variance_(true),
       frame_id_("gps")
-    {}
+    {
+      timer_ = create_wall_timer(1s, std::bind(&GPSDClientComponent::step, this));
+      start();
```

Comment:
Should initialise parameters before starting the timer:

```suggestion
    {
      start();
      timer_ = create_wall_timer(1s, std::bind(&GPSDClientComponent::step, this));
```